diff -ur Telephus/setup.py new/setup.py
--- Telephus/setup.py	2011-05-04 21:34:49.000000000 +0000
+++ new/setup.py	2011-05-04 21:38:23.000000000 +0000
@@ -2,7 +2,7 @@
 
 from distutils.core import setup
 setup(
-    name='telephus',
+    name='python-telephus',
     version='0.8.0~beta1',
     description='connection pooled, low-level client API for Cassandra in Twisted python',
     author='brandon@faltering.com',
